Gap3/Packages3/grim.html

```diff
@@ -7,7 +7,7 @@ <h3>
   GRIM: Groups of Rational and Integer Matrices
 </h3>
 <p>
-  Share package since release 3.4, about April 1997, communicated by Akos Seress (1958-2013).<br />
```

Comment:
We hardcoded that number `(1958-2013)` because before it was generated by mixer. Not sure what it means though so maybe we can remove it.

Reply:
it's his birth and death date. People inserted it when he died. But then e.g. Charles Sims who is mentioned also, also died, and no birth/death dates were added. So I decided it makes no sense for us to insert those everywhere.